src/test/resources/solidity/sol5/PrivacyInterface.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.5.9;
+
+interface PrivacyInterface {
```

Comment:
any reason to use the besu privacy contracts here? can you not use a generic greeter compiled with v5 solidity?

it feels wrong to use these contracts here....